extensions/windows-event-log/wel/UnicodeConversion.h

```diff
@@ -20,27 +20,27 @@
 
 #pragma once
 
-#include <string>
-
 #include <atlbase.h>
 #include <atlconv.h>
 
+#include <string>
+
 namespace org {
-  namespace apache {
-    namespace nifi {
-      namespace minifi {
-        namespace wel {
-          inline std::string to_string(const wchar_t* pChar) {
-            ATL::CW2A aString(pChar, CP_UTF8);
-            return std::string(aString);
-          }
+namespace apache {
+namespace nifi {
+namespace minifi {
+namespace wel {
+  inline std::string to_string(const wchar_t* pChar) {
+    ATL::CW2A aString(pChar, CP_UTF8);
+    return std::string(aString);
+  }
 
-          inline std::wstring to_wstring(const char* pChar) {
-            ATL::CA2W wString(pChar, CP_UTF8);
-            return std::wstring(wString);
-          }
-        } /* namespace wel */
-      } /* namespace minifi */
-    } /* namespace nifi */
-  } /* namespace apache */
+  inline std::wstring to_wstring(const char* pChar) {
+    ATL::CA2W wString(pChar, CP_UTF8);
+    return std::wstring(wString);
+  }
```

Comment:
These should still not be indented. Namespace contents are not indented.

Reply:
thanks, fixed in [95aeacd](https://github.com/martinzink/nifi-minifi-cpp/commit/95aeacd78262361de87609058f1f208b01e4fe3b)